### PR TITLE
Tmedia 741 byline global phrase

### PR DESCRIPTION
**Pull request by JackHowa**
- Move translations in lokalise from byline to global phrases 
- Run generate intl to generate the block changes
- goes with @malavikakoppula 's large promo #1411

### CODE REVIEW

File: blocks/overline-block/intl.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
 	"overline-sponsored-content": {
-		"de": "Werblicher Inhalt",
+		"de": "Sponsored Content",
```
Comment (JackHowa) [inline blocks/overline-block/intl.json]: interesting this looks like a regression

File: blocks/byline-block/intl.json [hunk 2]
```diff
@@ -1,22 +0,0 @@
-{
```
Comment (matthewroach) [inline blocks/byline-block/intl.json:1]: Removing this file will require updates to the code in this block to ensure it's pull the translation by the new key names now

File: blocks/shared-styles/intl.json [hunk 3]
```diff
@@ -21,28 +21,6 @@
 		"pt": "Vídeo",
 		"sv": "Video"
 	},
-	"byline.by-text": {
```
Comment (matthewroach) [inline blocks/shared-styles/intl.json]: This will mean current blocks (not migrated) will not have any translations for these phrases at the moment

Comment (JackHowa) [reply blocks/shared-styles/intl.json]: oh I thought that's what we wanted. will update

Comment (matthewroach) [reply blocks/shared-styles/intl.json]: Yeah - that's fine by me, but may cause some confusion when a developer comes to do a migration of a block still using code from Shared Styles - technically there are still consumers of this code until the last block is converted

File: blocks/byline-block/intl.json (hunk 2 above)
Comment (JackHowa) [reply blocks/byline-block/intl.json:1]: updated

File: blocks/shared-styles/intl.json (hunk 3 above)
Comment (JackHowa) [reply blocks/shared-styles/intl.json]: added back

**Comment by JackHowa** [comment]
interesting -- identity seems to have changed in lokalise. looks like something with connected as changed. it should just be "connect"

**Comment by JackHowa** [comment]
hopefully this will resolve @nschubach and @malavikakoppula's work

File: blocks/byline-block/intl.json (hunk 2 above)
Comment (matthewroach) [reply blocks/byline-block/intl.json:1]: I pushed an updated - as the byline block intl.json file is manually curated and the values are not part of any locale main file

Comment (JackHowa) [reply blocks/byline-block/intl.json:1]: nice

File: blocks/overline-block/intl.json (hunk 1 above)
Comment (matthewroach) [reply blocks/overline-block/intl.json]: We can take the existing values and add to lokalise then regenerate the locale files to avoid this regression

**Review by matthewroach — DISMISSED** [review]
We need to update locale to avoid the sponsored content regression

**Comment by nschubach** [comment]
> We need to update locale to avoid the sponsored content regression

@matthewroach : Are you referring to this which I added to `extra-large-promo-block`?
https://github.com/WPMedia/arc-themes-blocks/blob/f67cf98c4dc14f5d0e46ffa2a81fc00d36132f5d/blocks/global-phrases-block/intl.json#L134-L144

**Comment by matthewroach** [comment]
@nschubach - Yes, if we have those translation available we should be setting them in their respective locations correctly

File: blocks/overline-block/intl.json (hunk 1 above)
Comment (matthewroach) [reply blocks/overline-block/intl.json]: We need to fix these before merging

Comment (JackHowa) [reply blocks/overline-block/intl.json]: oh ok

Comment (JackHowa) [reply blocks/overline-block/intl.json]: it seems like lokalise may have an issue ... trying to see what's happened there

Comment (JackHowa) [reply blocks/overline-block/intl.json]: cool can manually update lokalise with this

Comment (JackHowa) [reply blocks/overline-block/intl.json]: updated to overline

**Comment by JackHowa** [comment]
@matthewroach feedback around sponsored content has been addressed. not sure how that happened on lokalise's side. updated to global phrases

File: blocks/subscriptions-block/intl.json [hunk 4]
```diff
@@ -77,15 +77,15 @@
...
-		"sv": "Connected as %{email}"
+		"de": "Connected",
+		"en": "Connected",
+		"es": "Connected",
+		"fr": "Connected",
+		"ja": "Connected",
+		"ko": "Connected",
+		"no": "Connected",
+		"pt": "Connected",
+		"sv": "Connected"
```
Comment (nschubach) [inline blocks/subscriptions-block/intl.json:88]: You said this broke something?

It's passed in Subs, ContactInfo: 
image

Comment (nschubach) [reply blocks/subscriptions-block/intl.json:88]: not sure it's necessary as is so I don't care either way.

File: locale/de.json [hunk 5]
```diff
@@ -1,448 +1,447 @@
...
+		"category-carousel.left-arrow-label": "Previous",
+		"category-carousel.slide-indicator": "Slide %{current} of %{maximum}"
+	},
+	"product-assortment-carousel-block": {
+		"product-assortment-carousel.aria-label": "Products",
+		"product-assortment-carousel.right-arrow-label": "Next",
+		"product-assortment-carousel.left-arrow-label": "Previous",
+		"product-assortment-carousel.slide-indicator": "Slide %{current} of %{maximum}"
+	}
+}
```
Comment (nschubach) [inline locale/de.json]:
```suggestion
}

```

**Review by nschubach — APPROVED** [review]

**Pull request merged**